github-actions/DotNet.GitHubAction/README.md

```diff
@@ -1,8 +1,36 @@
-<!--
+---
+languages:
+- csharp
+products:
+- dotnet
+- github
+page_type: sample
+name: "GitHub Actions with .NET"
+urlFragment: "create-dotnet-github-action"
+description: "A .NET 5 console application that contains the example from Create a GitHub Action with .NET tutorial."
+---
 
-    TODO:
+# Create a GitHub Action with .NET
 
-    1. Write the tutorial around this code in dotnet/docs
-    2. Write the README.md file content to light up the samples/browser experience
+This sample is a console application written in C# from the [Create a GitHub Action with .NETtutorial](https://docs.microsoft.com/dotnet/github/create-dotnet-github-action). The article gives an overview of what is required to create a .NET GitHub action.
```

Comment:
I think this folder will change. We discussed dotnet/fundamentals/devops as a new folder.